test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/project.json

```diff
@@ -16,6 +16,7 @@
     "Microsoft.NETCore.Platforms": "1.0.1-*",
     "xunit.runner.aspnet": "2.0.0-aspnet-*"
```

Comment:
Sure, which we'll do it in a follow up. This one gets net451 to build and run tests using dotnet.